VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleListEntryController.cs

```diff
@@ -116,6 +116,51 @@ public IHttpActionResult CreateLinks(webModel.ListEntryLink[] links)
             return StatusCode(HttpStatusCode.NoContent);
         }
 
+        [HttpPost]
+        [Route("links/bulkcreate")]
```

Comment:
Need to rename API method to the following path
`~/api/catalog/listentrylinks/bulkcreate`